src/js/SelectionControls/SelectionControl.d.ts

```diff
@@ -6,6 +6,7 @@ type Types = 'checkbox' | 'radio' | 'switch';
 export interface BaseSelectionControlProps extends Props {
   id: IdPropType;
   name: number | string;
+  label: string;
```

Comment:
Can you change this to optional and to `React.ReactNode`? One of the recent changes that have been added to selection controls is that either the `label` or the `aria-label` are required since there are some cases where a label shouldn't be displayed. A great example is the DataTable checkboxes